[2023-09-06T20:28:34.743-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: streamings-dag.twitter_crawler manual__2023-09-06T23:28:32.685455+00:00 [queued]>
[2023-09-06T20:28:34.753-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: streamings-dag.twitter_crawler manual__2023-09-06T23:28:32.685455+00:00 [queued]>
[2023-09-06T20:28:34.753-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-09-06T20:28:34.771-0300] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): twitter_crawler> on 2023-09-06 23:28:32.685455+00:00
[2023-09-06T20:28:34.773-0300] {standard_task_runner.py:57} INFO - Started process 8237 to run task
[2023-09-06T20:28:34.777-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'streamings-dag', 'twitter_crawler', 'manual__2023-09-06T23:28:32.685455+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/streamings-dag.py', '--cfg-path', '/tmp/tmpk6uguaqs']
[2023-09-06T20:28:34.778-0300] {standard_task_runner.py:85} INFO - Job 6: Subtask twitter_crawler
[2023-09-06T20:28:34.817-0300] {task_command.py:415} INFO - Running <TaskInstance: streamings-dag.twitter_crawler manual__2023-09-06T23:28:32.685455+00:00 [running]> on host dany.
[2023-09-06T20:28:34.876-0300] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='danielbcuadros@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='streamings-dag' AIRFLOW_CTX_TASK_ID='twitter_crawler' AIRFLOW_CTX_EXECUTION_DATE='2023-09-06T23:28:32.685455+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-06T23:28:32.685455+00:00'
[2023-09-06T20:28:34.877-0300] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-06T20:28:34.878-0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/dany/airflow/dags/../../Streaming-insights/venv/bin/python /home/dany/airflow/dags/../../Streaming-insights/etls/twitter-crawler.py']
[2023-09-06T20:28:34.883-0300] {subprocess.py:86} INFO - Output:
[2023-09-06T20:29:05.943-0300] {subprocess.py:93} INFO - 34
[2023-09-06T20:29:05.943-0300] {subprocess.py:93} INFO - 46
[2023-09-06T20:29:05.943-0300] {subprocess.py:93} INFO - Disney Plus conclu√≠do com sucesso!
[2023-09-06T20:29:05.943-0300] {subprocess.py:93} INFO - Traceback (most recent call last):
[2023-09-06T20:29:05.943-0300] {subprocess.py:93} INFO -   File "/home/dany/airflow/dags/../../Streaming-insights/etls/twitter-crawler.py", line 151, in <module>
[2023-09-06T20:29:05.943-0300] {subprocess.py:93} INFO -     crawler.main()
[2023-09-06T20:29:05.943-0300] {subprocess.py:93} INFO -   File "/home/dany/airflow/dags/../../Streaming-insights/etls/twitter-crawler.py", line 146, in main
[2023-09-06T20:29:05.943-0300] {subprocess.py:93} INFO -     self.export_html_pages()
[2023-09-06T20:29:05.944-0300] {subprocess.py:93} INFO -   File "/home/dany/airflow/dags/../../Streaming-insights/etls/twitter-crawler.py", line 126, in export_html_pages
[2023-09-06T20:29:05.944-0300] {subprocess.py:93} INFO -     with open(file_path, 'w') as json_file:
[2023-09-06T20:29:05.944-0300] {subprocess.py:93} INFO - FileNotFoundError: [Errno 2] No such file or directory: '../twitter_html/twitter_html_Disney_Plus.json'
[2023-09-06T20:29:05.969-0300] {subprocess.py:97} INFO - Command exited with return code 1
[2023-09-06T20:29:05.976-0300] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/dany/airflow/airflow-venv/lib/python3.10/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-09-06T20:29:05.978-0300] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=streamings-dag, task_id=twitter_crawler, execution_date=20230906T232832, start_date=20230906T232834, end_date=20230906T232905
[2023-09-06T20:29:06.001-0300] {standard_task_runner.py:104} ERROR - Failed to execute job 6 for task twitter_crawler (Bash command failed. The command returned a non-zero exit code 1.; 8237)
[2023-09-06T20:29:06.031-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-06T20:29:06.047-0300] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
